### PR TITLE
Adding a note about broken SAML SSO

### PR DESCRIPTION
**Pull request by datanerd**
### Why:

**Closes [issue link]**

### What's being changed:
I've added a note about SAML SSO breaking if organisation has been renamed. Part of SAML SSO configuration is a URL like `https://github.com/orgs/my-org-old-name`. [Here's an example](https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/github-tutorial#configure-azure-ad-sso) (Step no.4) for AzureAD case. When `my-org-old-name` has been renamed to `my-org-new-name`, those URLs in SAML SSO settings will no longer work and will need to be updated to `https://github.com/orgs/my-org-new-name`

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/setting-up-and-managing-organizations-and-teams/renaming-an-organization.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ After changing your organization's name:
 - Links to your previous organization profile page, such as `https://{% data variables.command_line.backticks %}/previousorgname`, will return a 404 error. We recommend you update links to your organization from other sites{% if currentVersion == "free-pro-team@latest" %}, such as your LinkedIn or Twitter profiles{% endif %}.
 - API requests that use the old organization's name will return a 404 error. We recommend you update the old organization name in your API requests.
 - There are no automatic [@mention](/articles/basic-writing-and-formatting-syntax/#mentioning-people-and-teams) redirects for teams that use the old organization's name.
+- If [SAML SSO](/github/setting-up-and-managing-organizations-and-teams/about-identity-and-access-management-with-saml-single-sign-on) has been configured for the organization, users will no longer be able to authenticate against your identity provider. To resolve this, update relevant parameters in SSO settings at your identity provider.
```
Comment (pierluigi) [inline content/github/setting-up-and-managing-organizations-and-teams/renaming-an-organization.md:38]:
```suggestion
- If [SAML single sign-on](/github/setting-up-and-managing-organizations-and-teams/about-identity-and-access-management-with-saml-single-sign-on) you must ensure the new organization name is reflected in the configuration of your identity provider (IdP), or users will no longer be able to authenticate.
```

Comment (pierluigi) [reply content/github/setting-up-and-managing-organizations-and-teams/renaming-an-organization.md:38]: Hopefully an improvement! 😉

**Review by pierluigi — DISMISSED** [review]
Awesome work @andrekolodochka ! Thanks for finding this and adding the missing documentation.

**Review by pierluigi — DISMISSED** [review]

**Review by rcableguytwc — DISMISSED** [review]

**Comment by mattpollard** [comment]
👋🏻 Thanks for adding this note, @andrekolodochka! And appreciate the revisions, @pierluigi 👏🏻

I revised the language a bit and fleshed some things out. I've also versioned the new unordered list item because the advice doesn't apply to products other than GitHub Enterprise Cloud. In addition to the GitHub Enterprise Server and GitHub AE not offering organization-level SAML SSO, the article(s) we've linked to isn't versioned for GitHub Enterprise Server or GitHub AE, and this was causing two checks to fail.

I'll merge this shortly, and it'll be live within 24 hours. Thanks again!

**Comment by andrekolodochka** [comment]
@mattpollard good point about showing it for Cloud plans only! Thanks for adding that!

**Review by mattpollard — APPROVED** [review]

**Pull request merged**